### PR TITLE
Added header form and welcome massage

### LINKED ISSUE #3: Add header with input field on title page
## Solution proposal

- [x] Create a header with a input field in it

- [x] Add a welcome message to the body

- [x] Add instructions below the welcome message

### PR DESCRIPTION
**Pull request by te4-andre-skvarc**
## What issue did you implement or fix?
Fixed #3 

## Summary
- Added header with form.
- Added welcome/info text in the body. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,6 +6,14 @@
     <title>Document</title>
 </head>
 <body>
+    <header><form>
+        <input type="text" placeholder="Username">
+        <button type="submit">Enter Username</button>
+    </form></header>
+    <h2>Welcome to teacher o matic</h2>
+    <p>Enter your user name in the header</p>
+
```
Comment (te4-adrian-almetun-smeds) [inline public/index.html:15]: Can this be placed in a wrapper?

```suggestion
<div class="wrapper">
    <h2>Welcome to teacher o matic</h2>
    <p>Enter your user name in the header</p>
</div>

```

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]
Nice one, just see if this should not be containerized.

**Review by te4-david-jensen — APPROVED** [review]
👍

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
:+1:

**Pull request merged**